packages/ui-chakra/src/base/Button/Button.tsx

```diff
@@ -2,9 +2,12 @@ import React from "react";
 
 import { Button as CKButton } from "@chakra-ui/react";
 
-type ButtonVariant = "primary" | "secondary" | "tertiary" | "accent" | "error";
+type ButtonVariant = "primary" | "secondary";
 type ButtonProps = {
   variant?: ButtonVariant;
+  width?: string;
+  marginTop?: string;
```

Comment:
Should the `<Button>` be responsible for handling the layout? or should its containing element be responsible for that instead?